src/cvpak.cpp

```diff
@@ -528,8 +528,9 @@ p       Cvnchars = sprintf( str, sif[lj][ipv], wid, ppos, 0);
 		for (i = 0; ; i++)
 		{
 			// more robust to give up HERE if i too big ?? 10-88 rob
-			if ( fabs(val) < maxfit		// if now might fit width
-			&&  cvdd( mfw-1, dfw) )  		// format it and see
+			if ((fabs(val) < maxfit		// if now might fit width
+			&&  cvdd( mfw-1, dfw)) ||  		// format it and see
+				std::isnan(val))            // guard against inf loop
```

Comment:
I stand by my prior comment that the nan case should be intercepted earlier.  Suitable output should be generated, e.g. "nan".  Maybe issue warning?  (Or maybe not ... printf() does not.  What does fmt do?)   Also, as suggested add one or more nan cases to the cvpak test to exercise the new capability.